### DISCUSSION TITLE
Chris H

### DISCUSSION THREAD

**Discussion by TheHammerMan00** [makerforgetech/modular-biped#61] [Introductions]
Hello everyone!

I'm building an inmoov robot but I wanted to work on something smaller while I have parts on order for the big guy.  I thought this project was perfect so I'm sitting here watching my first parts print while I introduce myself.  I can already feel the obsession setting in, lol

have you guys named your builds?

**Comment by Chompalot** [comment]
Hi Chris, the inmoov look ambitious. I started building a humanoid sized robot, but didn't get very far, just a head and laser cut torso. The actuators seemed to be a bit of a sticky wicket as I didn't want to shell out a lot of cash.
No I haven't though of naming the build yet, perhaps when I start getting some movement, so I can see a bit of character.
Cheers Simon.
